working/v3.0.0-draft2/authorization-code-flow/PAR-response-schema.json

```diff
@@ -0,0 +1,27 @@
+{
+    "title": "Version 1 of PAR response payload",
+    "$id": "http://apicentre.paymentsnz.co.nz/schemas/par-response-schema.json",
+    "type": "object",
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "additionalProperties": false,
+    "required": [
+      "request_uri",
+      "expires_in"
+    ],
+    "properties": {
+      "request_uri": {
+        "$id": "/properties/request_uri",
+        "type": "string",
+        "format": "uri",
+        "description": "A unique reference for the request object requiring authorization",
+        "examples": ["urn:ietf:params:oauth:request_uri:8qi1b0XAGrV9Ypv1cMyLAS_2AIchvqXx"]
+      },
+      "expires_in": {
+        "$id": "/properties/expires_in",
+        "type": "integer",
+        "description": "Expiration time on or after which the ID Token MUST NOT be accepted for processing.",
```

Comment:
```suggestion
        "description": "Expiration time on or after which the request object MUST NOT be accepted for processing.",
```

Reply:
When the JARM response including `request_uri` is issued, the `expires_in` claim isn't the validity of the request object (and certainly not ID token).  The `expires_in` is how long the issued `request_uri` is valid for use in the subsequent authorise request.  Will update the description.